scripts/load-people

```diff
@@ -78,6 +78,10 @@ sub db_connect {
 
     $gradd = $dbh->prepare("replace into gidredirect (gid_from, gid_to) values (?,?)");
     $grdelete = $dbh->prepare("delete from gidredirect where gid_from = ?");
+
+    $start_transaction = $dbh->prepare("START TRANSACTION");
```

Comment:
Same as other file - looks like you only want to turn off AutoCommit for the load_people function, if that's the behaviour you want.